tests/openebs-pool-container-failure_test.go

```diff
@@ -21,11 +22,12 @@ import (
 )
 
 var (
-	kubeconfig string
-	config     *restclient.Config
-	client     *kubernetes.Clientset
-	clientSet  *chaosClient.LitmuschaosV1alpha1Client
-	err        error
+	kubeconfig        string
+	config            *restclient.Config
+	client            *kubernetes.Clientset
+	clientSet         *chaosClient.LitmuschaosV1alpha1Client
+	err               error
```

Comment:
Add variables for labels and ns here : 
    - `cspPodLabels := "app=cstor-pool" `
    - `cspPodNs := openebs`
Use these variables at every places instead of hardcoded the things

Reply:
here it will not take ... I have defined the vars `cspPodLabels` `cspPodNs` after describe (global for all BDDs)